src/clean.js

```diff
@@ -14,6 +14,14 @@ function clean(ast) {
         : { kind: ast.kind, level };
     }
   }
+
+  if (ast.kind === "magic") {
```

Comment:
can you add a comment for both of these explaining we change case in the printer?